app/controllers/users/dossiers_controller.rb

```diff
@@ -106,13 +106,16 @@ def siret_informations
     update_current_user_siret!(siret)
 
     etablissement_attributes = SIRETService.fetch(siret, @facade.dossier)
+    Rails.logger.info("etablissement_attributes for siret #{siret}, #{etablissement_attributes.present?}")
```

Comment:
` +    Rails.logger.info("etablissement_attributes for siret #{siret} present? #{etablissement_attributes.present?}") `